### ISSUE TITLE
Import the resources from NLP-Progress semantic parsing collection

### ISSUE THREAD

**Issue by rajaswa** [semantic-parsing/semantic-parsing.github.io#1]
Import all the papers for `AMR` `DRS` `UCCA` parsing from: http://nlpprogress.com/english/semantic_parsing.html

1. Follow the [contributing guidelines](https://semantic-parsing.github.io/contributing.html)
2. Add all the extra resources: **code, website, paper, pre-print, etc.** for each paper
3. Assign proper **tags** to each paper
4. Allot each paper to appropriate [taxonomy](https://semantic-parsing.github.io/base-taxonomy/) by adding entries in the [taxonomy yml file](https://github.com/semantic-parsing/semantic-parsing.github.io/blob/source/_data/allamanistaxonomy.yml)

**Comment by Shreyas-Bhat** [comment]
Hello,
Are we supposed to make markdown files like these: https://github.com/semantic-parsing/semantic-parsing.github.io/tree/source/_publications? and will the format remain the same? 

What are tags and where do I add them?

Thank you

**Comment by rajaswa** [comment]
Yes, we are supposed to add a similar markdown file for each paper @Shreyas-Bhat. The format will remain the same. You can find the tags field in the markdown itself.
